Authentication/Login/View/LoginConfiguration.swift

```diff
@@ -13,7 +13,7 @@ import Foundation
 */
 public protocol LoginControllerTransitionDelegate {
```

Comment:
That could be an option too. My point is that this delegate should be only used to decide what to do with the login screen. Any other "independent" action that needs to be performed when the user logins should be done using the session service.